apps/data_handler/handlers/events/zklend/transform_events.py

```diff
@@ -82,31 +131,127 @@ def fetch_and_transform_events(self, from_address: str, min_block: int, max_bloc
+            event_data={
+                "liquidator": event_data.liquidator,
+                "user": event_data.user,
+                "debt_token": event_data.debt_token,
+                "debt_raw_amount": event_data.debt_raw_amount,
+                "debt_face_amount": event_data.debt_face_amount,
+                "collateral_token": event_data.collateral_token,
+                "collateral_amount": event_data.collateral_amount
+            }
+        )
+
+    def save_borrowing_event(self, event_name: str, block_number: int, event_data: BorrowingEventData) -> None:
+        """
+        Save a borrowing event to the database.
+        """
+        self.db_connector.create_borrowing_event(
+            protocol_id=self.PROTOCOL_TYPE,
+            event_name=event_name,
+            block_number=block_number,
+            event_data={
+                "user": event_data.user,
+                "token": event_data.token,
+                "raw_amount": event_data.raw_amount,
+                "face_amount": event_data.face_amount
+            }
+        )
+
+    def save_deposit_event(self, event_name: str, block_number: int, event_data: DepositEventData) -> None:
+        """
+        Save a deposit event to the database.
+        """
+        self.db_connector.create_deposit_event(
+            protocol_id=self.PROTOCOL_TYPE,
+            event_name=event_name,
+            block_number=block_number,
+            event_data={
+                "user": event_data.user,
+                "token": event_data.token,
+                "face_amount": event_data.face_amount
+            }
+        )
+    
+    def save_withdrawal_event(self, event_name: str, block_number: int, event_data: WithdrawalEventData) -> None:
+        """
+        Save a withdrawal event to the database.
+        """
+        self.db_connector.create_withdrawal_event(
+            protocol_id=self.PROTOCOL_TYPE,
+            event_name=event_name,
+            block_number=block_number,
+            event_data={
+                "user": event_data.user,
+                "token": event_data.token,
+                "amount": event_data.amount
+            }
+        )
+    
+    def save_collateral_enabled_disabled_event(self, event_name: str, block_number: int, event_data: CollateralEnabledDisabledEventData) -> None:
+        """
+        Save a collateral enabled/disabled event to the database.
+        """
+        self.db_connector.create_collateral_enabled_disabled_event(
+            protocol_id=self.PROTOCOL_TYPE,
+            event_name=event_name,
+            block_number=block_number,
+            event_data={
+                "user": event_data.user,
+                "token": event_data.token
+            }
+        )
+    
+    def save_repayment_event(self, event_name: str, block_number: int, event_data: RepaymentEventData) -> None:
+        """
+        Save a repayment event to the database.
+        """
+        self.db_connector.create_repayment_event(
```

Comment:
I can't see this method in db_connector, please add for each function which is missing